code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -98,7 +98,6 @@
 	stealth = TRUE
 	RegisterSignal(owner, COMSIG_MOVABLE_MOVED, .proc/handle_stealth)
 	handle_stealth()
-	ADD_TRAIT(owner, TRAIT_TURRET_HIDDEN, STEALTH_TRAIT)
```

Comment:
?

Reply:
no idea why thats there, it showed up after i pulled it from main tgmc. gona remove it.